Source/WebCore/rendering/RenderLayerModelObject.h

```diff
@@ -95,6 +96,9 @@ class RenderLayerModelObject : public RenderElement {
 
     // TODO: [LBSE] Add more cases here, as soon as we add the corresponding resources.
     RenderSVGResourceClipper* svgClipperResourceFromStyle() const;
+    RenderSVGResourceMasker* svgMaskerResourceFromStyle() const;
+    void paintSVGClippingMask(PaintInfo&) const;
+    void paintSVGMask(PaintInfo&, const LayoutPoint&) const;
```

Comment:
The necessity of 'adjustedPaintOffset' (the 2nd parameter) or the absence of it in `paintSVGClippingMask` should be revisited / clarified. IMHO one should spell out the parameter name here for clarity - if it doesn't violate style rules.

Reply:
I added the name, indeed it can't be deduced from the parameter type.